### ISSUE TITLE
Discuss users UX

### ISSUE THREAD

**Issue by ChShersh** [kowainik/issue-wanted#3]
I would like to have an ability for users to see their achievements depending on their open-source contribution. This is the biggest motivation for people to do something. So we should have an ability to login into our application through GitHub (in perfect case).

Also, I'm not sure that we should synchronise contributors past... Let's track everything starting from server start.

So I propose the following sync scheme:

* Sync all activity for user starting from server start and calculate all achivements
* Sync all achievements for users no often than in 4 hours (?) (or smarter strategy, depending on user activity?...)
* If user doesn't do anything for 30 days, stop syncing him (to not waste our resources)
* Have _force sync_ button in API

We should also assign points to every achievement and have ranking table...

**Comment by ChShersh** [comment]
Tags to consider:

`good first issue` `help wanted` `wanted` `up-for-grabs` `beginner-friendly`

**Comment by rashadg1030** [comment]
@chshersh I think the idea of an achievement board is cool. It's a good motivator. Along with all the tags @chshersh mentioned I think we should also include issues with `docs` or any other doc related tags to the list. It's not really programming, but fixing docs is a good way for programmers to get introduced to a codebase. I think the sync strategy that @chshersh proposed is good. By "server start" do you mean when they use `issue-wanted` for the first time they start off fresh? I think that's fine.

**Comment by chshersh** [comment]
> By "server start" do you mean when they use issue-wanted for the first time they start off fresh?

Yes, I meant that some hardcoded timestamp of the project start. However, I think it shouldn't be that difficult to fetch all user information (I hope).
